io.kopia.KopiaUI.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id": "io.kopia.KopiaUI",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "19.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "kopia-ui",
+    "separate-locales": false,
+    "rename-desktop-file": "kopia-ui.desktop",
+    "rename-icon": "kopia-ui",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=dri",
+        "--filesystem=host",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--own-name=org.kde.StatusNotifierItem-2-1"
+    ],
+    "modules": [
+        {
+            "name": "kopia-ui",
+            "buildsystem": "simple",
+            "build-commands": [
+                "ar x kopia-ui_*.deb",
+                "rm -f kopia-ui_*.deb",
+                "tar xf data.tar.xz",
+                "rm -f control.tar.gz data.tar.xz debian-binary",
+                "cp -r usr/* opt/* /app",
+                "rm -rf /app/share/icons/hicolor/1024x1024/",
+                "chmod -R a-s,go+rX,go-w /app/KopiaUI",
+                "sed -i 's|Exec=.*|Exec=kopia-ui %U|' /app/share/applications/kopia-ui.desktop",
+                "install kopia-ui.sh /app/bin/kopia-ui",
+                "install -Dm644 io.kopia.KopiaUI.appdata.xml /app/share/appdata/io.kopia.KopiaUI.appdata.xml"
+            ],
+            "sources": [
+                {
+                    "type": "file",
+                    "only-arches": [
+                        "x86_64"
+                    ],
+                    "url": "https://github.com/kopia/kopia-ui-release/releases/download/20200717.0.200337/kopia-ui_20200717.0.200337_amd64.deb",
+                    "sha256": "629ba5a7ce0d4e2fc076630c838b78b31ea6701aef5e63d1b18a8558adcf3039"
+                },
+                {
+                    "type": "script",
+                    "dest-filename": "kopia-ui.sh",
+                    "commands": [
+                        "export TMPDIR=\"$XDG_RUNTIME_DIR/app/$FLATPAK_ID\"",
+                        "exec zypak-wrapper /app/KopiaUI/kopia-ui \"$@\""
+                    ]
+                },
+                {
+                    "type": "file",
+                    "path": "io.kopia.KopiaUI.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
It is not clear to me what exactly the difference is. Should I add a `<launchable>` attribute for the `.desktop` file?